build_debian.sh

```diff
@@ -585,7 +579,13 @@ sudo du -hsx $FILESYSTEM_ROOT
 sudo mkdir -p $FILESYSTEM_ROOT/var/lib/docker
 sudo mksquashfs $FILESYSTEM_ROOT $FILESYSTEM_SQUASHFS -e boot -e var/lib/docker -e $PLATFORM_DIR
 
-scripts/collect_host_image_version_files.sh $TARGET_PATH $FILESYSTEM_ROOT
+sudo scripts/collect_host_image_version_files.sh $TARGET_PATH $FILESYSTEM_ROOT
```

Comment:
The post_run_buildinfo is run as root inside the script, maybe not necessary to add sudo here, but adding sudo should be good.

Reply:
without sudo, there is a error while copying the version files: 
/bin/cp: cannot create regular file 'target/versions/host-image/post-versions/versions-deb-buster-armhf': Permission denied